charts/schema-loading/Chart.yaml

```diff
@@ -11,7 +11,8 @@ keywords:
 - DLT
 - schema-loading
 home: https://scalar-labs.com/
-source: ["https://scalardl.readthedocs.io/en/latest/"]
+sources:
+  - https://scalardl.readthedocs.io/en/latest
```

Comment:
```suggestion
  - https://github.com/scalar-labs/scalardl
```

Reply:
Thank you for your suggestion.
Source URL updated 9f761b07464ca50ad3e370c84b66e99b6d3faeae

Reply:
I think it would be better if the following were included in the `sources`.
- https://github.com/scalar-labs/scalardl-schema-loader
- https://github.com/scalar-labs/scalardb

But `scalardl-schema-loader` is a private repo yet, So I'm not sure if it can be specified.

Reply:
Thank you @tei-k san.

I think it's good to use
* https://github.com/scalar-labs/scalardb/tree/master/tools/scalar-schema

@feeblefakie san,
Can we upload the source with the above URL?

Reply:
> https://github.com/scalar-labs/scalardb/tree/master/tools/scalar-schema

In many charts, the repo url was specified. I'm not sure if it's a good idea to specify a path. (Maybe it's OK!)

Reply:
Let's got with https://github.com/scalar-labs/scalardb/tree/master/tools/scalar-schema.